### PR TITLE
Rework the "About the Java Track" page - #347

### PR DESCRIPTION
**Pull request by RhuanKrum**
Page re-written to be more friendly and accessible.
* This pull request fixes the issue #347.

Ps. This is my rework proposal for this page. Feel free to suggest changes or improvements. Thanks!

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,35 +1,7 @@
+[Java](https://go.java/index.html) is among the most popular available programming languages, thanks to its versatility and compatibility. It is widely used for software development, mobile applications, and large systems development.
```
Comment (FridaTveit) [inline docs/ABOUT.md]: Maybe `and developing larger systems`. I also don't think there should be a comma before the `and`. But I'm no expert on comma rules, so feel free to correct me if I'm wrong :)

Comment (FridaTveit) [reply docs/ABOUT.md]: Also, as part of the Exercism documentation guidelines, each sentence should start on a new line :)

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,35 +1,7 @@
...
 
-[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) is a general-purpose computer programming language that is: 
- 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing) 
- 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
- 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+Besides it is not that young as the fresh languages out there, Java which was born in 1995 and is maintained by [Oracle](https://www.oracle.com/index.html) was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists as it must be execute through a cross-platform compatible [Java Virtual Machine - JVM](https://en.wikipedia.org/wiki/Java_virtual_machine). Android apps are also developer using Java, since the [Android Operating System](https://en.wikipedia.org/wiki/Android_(operating_system)) runs on a Java language environment.
```
Comment (FridaTveit) [inline docs/ABOUT.md]: These sentences don't quite make sense in English. I would suggest something like:
"
Java was born in 1995 and is maintained by [Oracle](https://www.oracle.com/index.html). Despite the fact that it isn't as young as some of the fresh languages out there, Java is still really popular now. It was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists because Java is executed on a cross-platform compatible [Java Virtual Machine - JVM](https://en.wikipedia.org/wiki/Java_virtual_machine). Android apps are also developed using Java, since the [Android Operating System](https://en.wikipedia.org/wiki/Android_(operating_system)) runs on a Java language environment.
"

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1,35 +1,7 @@
...
 
-Java applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Java virtual machine](https://en.wikipedia.org/wiki/Java_virtual_machine) (JVM) regardless of computer architecture.
+The Java community size is huge (GitHub for example has over 1.5 million Java projects, also mentioning that Java has the second larger community in [StackOverflow](https://stackoverflow.com/questions/tagged/java))! This is important because the larger a programming language community is, the more support you'd be likely to get.
```
Comment (FridaTveit) [inline docs/ABOUT.md]: This also doesn't really make sense in English. You could slightly rephrase it to:
"
The Java community is huge. GitHub for example has over 1.5 million Java projects. It's also worth mentioning that Java has the second largest community in [StackOverflow](https://stackoverflow.com/questions/tagged/java)! This is important because the larger a programming language community is, the more support you'd be likely to get.
"

File: docs/ABOUT.md [hunk 4]
```diff
@@ -1,35 +1,7 @@
...
-- JAVA IS EVERYWHERE. 
-It is one of the most sought after programming languages.
-It is a standard for enterprise software, web-based content, games and mobile apps, as well as the [Android operating system](https://developer.android.com/index.html). 
-
-- [Write once, run anywhere](https://en.wikipedia.org/wiki/Write_once,_run_anywhere) policy: Programs written in Java are compiled into machine-independent bytecodes, so it can run on any Java platform.
-Basically, Java is [Platform Independent](https://en.wikipedia.org/wiki/Cross-platform#Platform-independent_software)
-
-- Java has rich APIs:
-Java has built-in [application programming interfaces](https://en.wikipedia.org/wiki/Application_programming_interface) (APIs) for I/O, networking, data structures, concurrency, and many other functions.
-Standard libraries can be supplemented by open source libraries like Apache Commons, Google Guava, Netty and others.
```
Comment (FridaTveit) [inline docs/ABOUT.md]: I think this bit about the Java API is quite important. Java has a really good API with really good documentation which makes it a lot easier to use. So it might be worth mentioning something about that, possibly linking to the API docs as well: https://docs.oracle.com/javase/8/docs/api/ :)

File: docs/ABOUT.md [hunk 5]
```diff
@@ -1,35 +1,7 @@
...
-
-- Standardization:
-If you need to write a web/enterprise app, or you need data persistence or you need to do cryptography, no problem!
-Java is all you require.
-There are many standard tools/practices that extend the Java ecosystem, thanks to the [Java Community Process](https://www.jcp.org/en/introduction/overview) (JCP)!
+From laptops to datacenters, game consoles to scientific supercomputers, cell phones to the Internet, [Java is everywhere](https://en.wikipedia.org/wiki/Write_once,_run_anywhere)! There are lots of applications and websites that will not work unless you have Java installed (yes, we are talking about that lovely orange icon who always ask to update itself time to time!).
```
Comment (FridaTveit) [inline docs/ABOUT.md]: I would take out this last sentence. I'm not sure it adds anything very useful and the previous sentence is a really good one to end on :)

File: docs/ABOUT.md (hunk 1 above)
Comment (RhuanKrum) [reply docs/ABOUT.md]: I fixed the English mistakes as you suggested. Sorry about that.

I didn't find the guideline mentioning I should start each sentence in on a new line :(
Should it be here? https://github.com/exercism/docs/blob/master/language-tracks/documentation/introductory-copy.md

I adapted my file now by following the other pages, so I hope it is in the correct format now :)

File: docs/ABOUT.md (hunk 2 above)
Comment (RhuanKrum) [reply docs/ABOUT.md]: I fixed the English mistakes as you suggested. 
Again, sorry for that. I'll take better care of this kind of situations :(

File: docs/ABOUT.md (hunk 4 above)
Comment (RhuanKrum) [reply docs/ABOUT.md]: I added a paragraph about the APIs as you suggested. My bad for not including it before, thanks for that!

File: docs/ABOUT.md (hunk 5 above)
Comment (RhuanKrum) [reply docs/ABOUT.md]: Agreed! I'll take care of reading it several times before applying it next time. :)

File: docs/ABOUT.md (hunk 1 above)
Comment (Smarticles101) [reply docs/ABOUT.md]: @FridaTveit the last comma before the and, is usually known as a serial or oxford comma. There's quite a bit of controversy around it. Grammarly has a short blog post about it [here](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)

**Comment by RhuanKrum** [comment]
File updated! could you kindly review it again, please?

About making commits into the master branch, I had a branch where I was working on, but in some point I got a bit confused and commited in the master branch by mistake without noticing it. I'll take better care about it too!

Thanks!

File: docs/ABOUT.md (hunk 1 above)
Comment (FridaTveit) [reply docs/ABOUT.md]: @RhuanKrum I remember there was a long discussion about it in the `docs` repo I think. But saying that, I can't find it now! So it's probably not very important :)

Thanks @Smarticles101 :)
